be/src/exprs/string_functions.cpp

```diff
@@ -79,6 +79,13 @@ StringVal StringFunctions::right(
     return substring(context, str, IntVal(pos), len);
 }
 
+IntVal StringFunctions::ends_with(
```

Comment:
1. should check if input str and suffix is null.
2. return value is a BoolVal?
3. better add a unit test for this new added function
You can reference [here](https://github.com/apache/incubator-doris/pull/2731/files#diff-5e4308868edc8f5148f449c84f8428ffR281)